### DISCUSSION TITLE
Release planning

### DISCUSSION THREAD

**Discussion by hamza-fadil** [victorbalssa/abacus#298] [General]
Hello Victor, 
Do you have a release planning ? When are you planning to push 0.17 please ?
Any features you have in mind for the future ?
Thank you

**Comment by victorbalssa** [comment]
Hi @hamza-fadil, I've just sent the new OTA update with the last develop commit included.
I'm trying to follow the Roadmap: https://github.com/users/victorbalssa/projects/2

Feel free to propose new features, I will review them as soon as possible 🚀

**Reply by victorbalssa** [reply]
If you are working on something just create the Feature Request issue and assign it to you.

**Reply by hamza-fadil** [reply]
Thank you, I'll try to help as much as possible. Great app

**Comment by drewzeee** [comment]
Looks like iOS App Store dropped this app?

**Reply by victorbalssa** [reply]
It should be back now.
